package.json

```diff
@@ -25,6 +25,7 @@
     "ent": "^2.2.0",
     "eslint": "^4.19.1",
     "eslint-config-airbnb": "^16.1.0",
+    "eslint-config-prettier": "^2.9.0",
```

Comment:
We already have `eslint-plugin-prettier`. Why do we need to also have `eslint-config-prettier`?